src/components/ScrollToTop/ScrollToTop.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import { FaAngleUp } from 'react-icons/fa';
+import { useColorMode } from "@chakra-ui/react";
+import cn from "classnames";
+
+import '../../App.css';
+import styles from "../../layouts/Main/Main.module.css";
+
+const ScrollToTop = () => {
+    const { colorMode } = useColorMode();
+    const [showTopBtn, setShowTopBtn] = useState(false);
+    useEffect(() => {
```

Comment:
new lines between purpose-specific code-blocks and functions definition improve reading